tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -35,10 +37,12 @@ Verify Model Can Be Deployed Via UI
     [Tags]    Sanity    Tier1
     ...    ODS-1921
     Open Model Serving Home Page
-    # Verify No Models Are Present
-    Click Button    Create server
-    Wait Until Page Contains    Data science projects
-    Wait Until Page Contains Element    //button[.="Create data science project"]
+    Clean Up Model Serving Page
+    Clean Up DSP Page
+    Open Model Serving Home Page
+    Try Opening Create Server
+    SeleniumLibrary.Wait Until Page Contains    Data science projects
+    SeleniumLibrary.Wait Until Page Contains Element    //button[.="Create data science project"]
```

Comment:
I think you could replace both lines with  Wait for RHODS Dashboard to Load

Reply:
This is not only about the page loading, but also about finding the expected element(s)

Reply:
there's no such variable in this suite

Reply:
you can import it from DS Projects resources